diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/FileSize.kt

```diff
@@ -77,6 +77,6 @@ class FileSize : Rule("file-size") {
 
     class FileSizeConfiguration(config: Map<String, String>) : RuleConfiguration(config) {
         val maxSize = config["maxSize"]?.toLongOrNull() ?: MAX_SIZE
-        val ignoreFolders = config["ignoreFolders"]?.split(IGNORE_FOLDERS_SEPARATOR) ?: IGNORE_FOLDER
+        val ignoreFolders = config["ignoreFolders"]?.replace("\\s+".toRegex(),"")?.split(IGNORE_FOLDERS_SEPARATOR.toRegex()) ?: IGNORE_FOLDER
```

Comment:
Just a side note, in kotlin there are triple-quoted strings, where you don't need to escape special symbols. For such small regex it's fine, but for longer ones triple-quoted strings can be more readable.